source/_integrations/dynalite.markdown

```diff
@@ -53,17 +52,17 @@ name:
   type: string
   default: dynalite
 active:
-  description: Actively query network. When starting, it will query all devices for their current status, and also will send queries when some changes are in progress (e.g., lights dimming or covers moving). Better experience but creates more load on the Dynalite network. 
+  description: Actively query network. When starting, it will query all devices for their current status, and also will send queries when some changes are in progress (e.g. lights dimming or covers moving). Better experience but creates more load on the Dynalite network. Value can be 'on', 'off', our 'init', where 'init' will only send queries during the initial init of Home Assistant
   required: false
-  type: boolean
+  type: [boolean, string]
   default: false
 polltimer:
-  description: Polling interval for devices in transition. Value in seconds. When devices are in transition (e.g., a light fading), it will ask for a new state every X seconds until it is at the target level. Only relevant when active is set.
+  description: Polling interval for devices in transition. Value in seconds. When devices are in transition (e.g., a light fading), it will ask for a new state every X seconds until it is at the target level. Only relevant when active is set to 'on'.
```

Comment:
```suggestion
  description: "Polling interval for devices in transition. Value in seconds. When devices are in transition (e.g., a light fading), it will ask for a new state every X seconds until it is at the target level. Only relevant when active is set to `on`."
```